container/bitvector/bitvector_test.go

```diff
@@ -19,6 +20,7 @@ func (s *BitVectorSuite) SetUpTest(c *C) {
 	data := []byte{0xF0, 0x0F}
 	length := 12
 	s.vector = NewBitVector(data, length)
+	s.lVector = NewBitVector(data, 16)
```

Comment:
vector & lVector shouldn't share the same byte array.  make a copy of the data

Reply:
Sure!